LICENSE

```diff
@@ -186,7 +186,8 @@ APPENDIX: How to apply the Apache License to your work.
    same "printed page" as the copyright notice for easier
    identification within third-party archives.
 
-Copyright [yyyy] [name of copyright owner]
+Copyright 2010-2015 Chef Software Inc.
+Copyright 2014-2015 Joshua Timberman
```

Comment:
Fwiw, this stays in the boilerplate of the LICENSE, since this is text one would put in headers of source files and the like for work that is apache licensed.